### PR TITLE
Acrolinx 9/11

### PR DESCRIPTION
**Pull request by gewarren**

### CODE REVIEW

File: docs/standard/garbage-collection/implementing-disposeasync.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ helpviewer_keywords:
...
 
-It is typical when implementing the <xref:System.IAsyncDisposable> interface that classes will also implement the <xref:System.IDisposable> interface. A good implementation pattern of the <xref:System.IAsyncDisposable> interface is to be prepared for either synchronous, or asynchronous dispose. All of the guidance for implementing the dispose pattern also applies to the asynchronous implementation. This article assumes that you're already familiar with how to [implement a Dispose method](implementing-dispose.md).
+It is typical when implementing the <xref:System.IAsyncDisposable> interface that classes will also implement the <xref:System.IDisposable> interface. A good implementation pattern of the <xref:System.IAsyncDisposable> interface is to be prepared for either synchronous or asynchronous dispose. All of the guidance for implementing the dispose pattern also applies to the asynchronous implementation. This article assumes that you're already familiar with how to [implement a Dispose method](implementing-dispose.md).
```
Comment (gewarren) [inline docs/standard/garbage-collection/implementing-disposeasync.md:19]: Removed comma between "synchronous" and "or asynchronous".

File: docs/standard/garbage-collection/implementing-disposeasync.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ public async ValueTask DisposeAsync()
...
+> One primary difference in the async dispose pattern compared to the dispose pattern, is that the call from <xref:System.IAsyncDisposable.DisposeAsync> to the `Dispose(bool)` overload method is given `false` as an argument. When implementing the <xref:System.IDisposable.Dispose?displayProperty=nameWithType> method, however, `true` is passed instead. This helps ensure functional equivalence with the synchronous dispose pattern, and further ensures that finalizer code paths still get invoked. In other words, the `DisposeAsyncCore()` method will dispose of managed resources asynchronously, so you don't want to dispose of them synchronously as well. Therefore, call `Dispose(false)` instead of `Dispose(true)`.
```
Comment (gewarren) [inline docs/standard/garbage-collection/implementing-disposeasync.md:52]: Replace ; with , after "however".

**Comment by gewarren** [comment]
Ping @IEvangelist .

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**